keras_rs/src/utils/pairwise_loss_utils.py

```diff
@@ -1,9 +1,8 @@
-from typing import Callable, Optional
+from typing import Callable
```

Comment:
I'm not a huge fan of the concept of a `utils` folder in general, and I typically prefer the utils files to be in the same folder as where it's used. So for instance, I think this could be in `keras_rs/src/losses`.

What do you think? Note that keras itself is not consistent on this.